kyototycoon/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,27 @@
+name: Kyoto Tycoon
+
+files:
+  - name: kyototycoon.yaml
```

Comment:
I think the `auto_conf.yaml` file should be specified here too

Reply:
Oh, good catch :+1: